src/System.Net.Sockets/src/System/Net/Sockets/SocketPal.Windows.cs

```diff
@@ -802,7 +802,7 @@ public static SocketError Select(IList checkRead, IList checkWrite, IList checkE
             return (SocketError)socketCount;
         }
 
-        public static SocketError Shutdown(SafeCloseSocket handle, SocketShutdown how)
+        public static SocketError Shutdown(SafeCloseSocket handle, bool isConnected, SocketShutdown how)
         {
             SocketError err = Interop.Winsock.shutdown(handle, (int)how);
```

Comment:
It does not produce a warning: the C# compiler does not warn on unused parameters, only unused variables. I'll go ahead and add a comment.